### PR TITLE
fix(trading): add arbiscan links

### LINKED ISSUE #6553: Approve asset TX toast fires off to ethereum wrongly
## The Chore

The toast for an approve TX on Arbitrum gives you a link to Etherscan

## Tasks

- [ ]
- [ ]

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #6553 

# Description ℹ️

- [x] add Arbiscan links
- [x] fix Arbiscan links on approval 

# Demo 📺

image

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

File: libs/deposits/src/lib/approve-notification.tsx [hunk 1]
```diff
@@ -166,11 +171,19 @@ const ApprovalTxFeedback = ({
...
-  const txLink = tx.txHash && (
-    <EtherscanLink sourceChainId={chainId} tx={tx.txHash}>
-      {t('View on Etherscan')}
-    </EtherscanLink>
-  );
+  const chain = getChainName(chainId);
+
+  const txLink =
+    tx.txHash &&
+    (chain.includes('Ethereum') ? (
```
Comment (edd) [inline libs/deposits/src/lib/approve-notification.tsx:178]: **nitpick (non-blocking)**:  Use a chain ID lookup would probably be more robust than relying on the string. Not worth changing as bridges are not being added regularly.

**Review by edd — APPROVED** [review]
Approved, with a comment about unifying this with the explorer implementation in future

File: apps/governance/.env.mainnet [hunk 2]
```diff
@@ -5,6 +5,7 @@ NX_VEGA_URL=https://api.vega.community/graphql
 NX_VEGA_NETWORKS='{"DEVNET":"https://dev.governance.vega.xyz","TESTNET":"https://governance.fairground.wtf","MAINNET":"https://governance.vega.xyz"}'
 NX_ETHEREUM_PROVIDER_URL=https://mainnet.infura.io/v3/3ba145ccc2884bcd91213d8dc989ca76
 NX_ETHERSCAN_URL=https://etherscan.io
+NX_ARBISCAN_URL=https://arbiscan.io
```
Comment (mattrussell36) [inline apps/governance/.env.mainnet:8]: We already have two places defining an arbiscan root url. Now this adds a third. Probably not something to fix right now but I'd like to unify this in a .ts file which can be indexed into with a chain id

File: libs/environment/src/components/arbitrum-link.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import { ExternalLink } from '@vegaprotocol/ui-toolkit';
```
Comment (mattrussell36) [inline libs/environment/src/components/arbitrum-link.tsx:1]: This component is not required. EtherscanLink has already been modified to handle other chains. (We should rename it later though)

**Review by mattrussell36 — CHANGES_REQUESTED** [review]

File: apps/governance/.env.mainnet (hunk 2 above)
Comment (mattrussell36) [reply apps/governance/.env.mainnet:8]: Based on the fact that the EtherscanLink component already handles other chains we don't need this new ENV var at all

**Comment by daro-maj** [comment]
I can not start the app
image

**Pull request closed without merging**